### DISCUSSION TITLE
Easy Set Up Guide or How-To for Completely New Contributors

### DISCUSSION THREAD

**Discussion by BluePraise** [WordPress/developer-blog-content#70] [Topic Ideas]
It is hard to get started for completely new contributors. I would suggest a very practical guide on how to set up a local environment, how to use Trac, and other elements that come with this. 

Speaking from my own experience, these are things I struggle with: 
- How to I set up a local environment? (Why could I not find it faster, sooner, easier?) 
- Do I have the correct coding conventions?  (Why could I not find it faster?)
- I want to help with testing. How do I start?

If there is already such a practical guide, I suggest a numbered list with references.

### Context
I have been studying [The Contributor Handbook](https://make.wordpress.org/core/handbook/contribute/) I am a beginner in the group, but my experience as a developer tells me to not just dive in with Git/SVN/PR's etc when there is a running development flow. It would be good if we have some hands-on experiences or explanations from the official source.

**Comment by bph** [comment]
Thank you for the topics idea, but I would think it would be best to connect with the team [during various meetings](https://make.wordpress.org/meetings/) and start asking questions in their channels. Each team has their own onboarding process. 
The Gutenberg editor has [a separate contributor guide](https://developer.wordpress.org/block-editor/contributors/)

These are all excellent questions. 
Asking in the Team's Slack channel where most of the contributors hangout would be probably the first step. 

In opposite to the make.WordPress.org the target audience for the developer blog are not contributors, but developers who don't frequent the make blogs to learn about developer relevant changes to the software. Those are the millions of freelancers, and agency developer, as well as product developers for plugins and themes.

**Comment by mburridge** [comment]
c/f - https://wordpress.slack.com/archives/C02QB2JS7/p1676369716883809
